layouts/partials/navbar.html

```diff
@@ -58,7 +58,7 @@
         {{ if and (eq $has_cfa 1) (not (or (eq .Site.Params.hide_cfa_same_page "true") (eq $is_cfa_page 1) (eq $hide_call_for_action true))) }}
           <div class="{{ $.Site.Params.call_for_action_wrapper_classes | default "float-right hidden-xs" }}" id="btn-call-for-action">
             <div class="btn-starter">
-              <a class="btn btn-secondary margin-0" href="https://start.jakarta.ee/">Starter</a>
+              <a class="btn btn-secondary margin-0" href="https://start.jakarta.ee/">{{ .Site.Params.call_for_starter_action_text }}</a>
```

Comment:
@kazumura I see that you are copying the solution from the call_for_action_text. 

I want to share that the the best practice in hugo is to translate a string using with i18n: https://gohugo.io/content-management/multilingual/

This would require us to create a new file in the https://github.com/jakartaee/jakarta.ee/tree/src/i18n folder for this new Japanese translation. This will be needed to fully translate the site.

When creating a new translatable string, we also need to make sure that we add an entry in the default language which is English.

While your solution is not completely wrong. I felt it would be a good idea to raise this issue early as I know you plan to make many translations in short term.

Reply:
@chrisguindon 
I have several experiences about multi language in Hugo with JakartaOne Livestream website, and
I know the file tree structure and i18n folder in Hugo.
This PR is only minmum changes as the first step.
We will create i18n/ja.toml after this PR.

Reply:
@kazumura My feedback is quite simple. 

I would appreciate it if you would be open to accepting feedback so that we can collaborate!

Reply:
@chrisguindon 
I give up "Starter" for now.